components/StackedBarChart.tsx

```diff
@@ -38,53 +37,48 @@ function StackedBarChart({ data }: StackedBarChartProps) {
 
     // TODO -- what's the shape of retained.primaryCharges?
     // note: the type of the key is really a number
-    let primaryCharges: Record<string, string> = {}
+    let pcSet = new Set<string>()
 
-    console.log('num of cases\n', Object.keys(data).length)
+    console.log('num of cases\n', cases.length)
+
+    cases.forEach((c) => {
+        pcSet.add(c.charges[0].offense_type_desc)
```

Comment:
As per Nate's comments in [this thread](https://open-austin.slack.com/archives/C043L6HHH17/p1670556293155349), it sounds like we can generally treat the first charge as the "primary charge".

We probably want to pre-compute this value and add an endpoint (file).